CHANGELOG.md

```diff
@@ -1,410 +1,31 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
-
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## Release 10.0.0 (2020-06-03)
-
-[Changes since v10.0.0-mongodb.0](https://github.com/realm/realm-studio/compare/v10.0.0-mongodb.0...v10.0.0)
+## Release 10.0.0-beta.1 (2020-06-03)
```

Comment:
Should this be beta.1? I'm currently suggesting that we release this as v10.0.0, without a prerelease-id but on a separate channel to avoid v3 upgrading to it.
If you're sure about this, we need to update package.json and package-lock.json accordingly.